.buildkite/scripts/backport_branch.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+source .buildkite/scripts/common.sh
+
+set -euo pipefail
+
+if buildkite-agent meta-data exists BASE_COMMIT; then
+  BASE_COMMIT="$(buildkite-agent meta-data get BASE_COMMIT)"
+else
+  BASE_COMMIT=${BASE_COMMIT:-""}
+fi
+
+if buildkite-agent meta-data exists PACKAGE_NAME; then
+  PACKAGE_NAME="$(buildkite-agent meta-data get PACKAGE_NAME)"
+else
+  PACKAGE_NAME=${PACKAGE_NAME:-""}
+fi
+
+if buildkite-agent meta-data exists PACKAGE_VERSION; then
+  PACKAGE_VERSION="$(buildkite-agent meta-data get PACKAGE_VERSION)"
+else
+  PACKAGE_VERSION="${PACKAGE_VERSION:-""}"
+fi
+
+if buildkite-agent meta-data exists REMOVE_ALL_PACKAGES; then
+  REMOVE_ALL_PACKAGES="$(buildkite-agent meta-data get REMOVE_ALL_PACKAGES)"
+else
+  REMOVE_ALL_PACKAGES="${REMOVE_ALL_PACKAGES:-"false"}"
+fi
+
+if [[ -z "$PACKAGE_NAME" ]] || [[ -z "$PACKAGE_VERSION" ]]; then
+  buildkite-agent annotate "The variables **PACKAGE_NAME** or **PACKAGE_VERSION** aren't defined, please try again" --style "warning"
+  exit 1
+fi
+
+FULL_PACKAGE_NAME="${PACKAGE_NAME}-${PACKAGE_VERSION}"
+FULL_ZIP_PACKAGE_NAME="${PACKAGE_NAME}-${PACKAGE_VERSION}.zip"
+SOURCE_BRANCH="main"
+BACKPORT_BRANCH_NAME="backport-${PACKAGE_NAME}-${PACKAGE_VERSION}"
+PACKAGES_FOLDER_PATH="packages"
+
+isPackagePublished() {
+  local packageZip=$1
+  local responseCode=$(curl -s -o /dev/null -w "%{http_code}" "https://package-storage.elastic.co/artifacts/packages/${packageZip}")
```

Comment:
Nit. Separate commands to avoid hidden errors:

```suggestion
  local responseCode
  responseCode=$(curl -s -o /dev/null -w "%{http_code}" "https://package-storage.elastic.co/artifacts/packages/${packageZip}")
```